code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -123,7 +123,7 @@
 	drink_icon = "poisonberryjuice"
 	drink_name = "Glass of poison berry juice"
 	drink_desc = "A glass of deadly juice."
-	taste_message = "berry juice"
+	taste_message = "rotten berries"
```

Comment:
Feels like an unneeded nerf to an already underutilized ingredient.

Reply:
I'm not sure it can be called a nerf when it's only mechanical purpose is printing text to the chat.

Reply:
Well, now you can tell normal berry juice from poison berry juice by taste, which wasn't the case earlier.